Lib/idlelib/idle_test/run_coverage.py

```diff
@@ -0,0 +1,135 @@
+import argparse
+import os
+import pathlib
+import subprocess
+import sys
+import textwrap
+
+
+def main():
+    idlelib_path = find_idlelib_path()
```

Comment:
```suggestion
    idlelib_path = pathlib.Path(idlelib.__file__).parent
```

Is this not much simpler?

Reply:
Yes, much!

But I'd like the script to not assume that it's run in such a way that "import idlelib" will find the correct version of the module. That is, I don't want anyone accidentally using their system python3 to run the script and end up testing the version of idlelib in their system Python's copy of the stdlib.

That's why I've gone to some length to avoid that here. I'll also replace the use of `sys.executable`, now that I think of it.

Reply:
That makes more sense now you have explained, though I don't see how this is any different to someone using their system install to run the test suite (no workarounds are done to allow for that).

Reply:
As it is, I see this more as a script (which could be run with any Python interpreter). It is somewhat different from `python -m test` since it is not called with `python -m`.

Also, I expect this to be run from the `Lib/idlelib` or `Lib/idlelib/idle_test` directories often enough, so not having to type `../../../python` is nice :)

Reply:
Considering this, I should add a shebang line...

Reply:
... and make it executable.

Done.